docs/file-response-references.md

```diff
@@ -14,7 +14,7 @@ In this example, you can see the file response when the SDK was set to `useFileR
   zipCode: '8032',
   city: 'Zürich',
   country: 'Switzerland',
-  picture: 'https://media.jobs.ch/media/280d8fce-efc3-42f5-9c06-606aba92e2d3/cv-ann-boile.pdf?token=8d62',
```

Comment:
ann-boile.png

Reply:
The SDK's response does not deliver a file name, so this should be ok.